src/coreclr/jit/emitxarch.cpp

```diff
@@ -6148,9 +6259,9 @@ void emitter::emitIns_AR_R(instruction ins, emitAttr attr, regNumber reg, regNum
 //
 void emitter::emitIns_S_R_I(instruction ins, emitAttr attr, int varNum, int offs, regNumber reg, int ival)
 {
-    // This is only used for INS_vextracti128 and INS_vextractf128, and for these 'ival' must be 0 or 1.
-    assert(ins == INS_vextracti128 || ins == INS_vextractf128);
-    assert((ival == 0) || (ival == 1));
+    assert(IsSSEOrAVXInstruction(ins));
```

Comment:
earlier this assert just covered few instructions, but not it covers all the SSE/AVX instructions?

Reply:
Previously we could only hit this with those two instructions. Now we can hit it for several instructions, but they should only be SSE/AVX instructions. We don't check the "exact" list for most other paths so I opted to not here as well.